blocks/bricks/sequence_generators.py

```diff
@@ -290,38 +290,80 @@ def readout(self, **kwargs):
         pass
 
 
-@add_metaclass(ABCMeta)
-class Readout(AbstractReadout):
+class Readout(AbstractReadout, Initializable):
     """Readout brick with separated emitting and feedback parts.
 
     Parameters
     ----------
+    source_names : list
+        A list of the source names (outputs) that are needed for the
+        readout part e.g. ``['states']`` or ``['states', 'glimpses']``.
```

Comment:
There used to be a double-usage of term glimpse. Documentation of `attention.py` introduces it a name for a deliverable of an attention mechanism. Also the only available attention mechanism had one glimpse named "glimpse". I renamed the latter to stop this confusing collision. 

The 30 times you are talking about are most likely usages in the first sense, and those are okay. But there you refer to a name of a particular glimpse, which is since #420 called 'weighted_averages`.